src/tools/rbd/action/ObjectMapCheck.cc

```diff
@@ -0,0 +1,74 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
```

Comment:
This is for the cli -- most of the related commands are grouped into the same file.